src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -340,7 +340,22 @@ public PostGobiOrdersHelper(HttpClientInterface httpClient,
 
   public CompletableFuture<String> lookupMock(String data) {
     logger.info("Mocking the data lookup for: " + data);
-    return CompletableFuture.completedFuture(UUID.randomUUID().toString());
+    return CompletableFuture.completedFuture(UUID.randomUUID()
+      .toString());
+  }
+
+  public CompletableFuture<String> separateISBNQualifier(String data) {
+    if (data.split("\\s+").length > 1) {
+      return completedFuture(data.split("\\s+")[1]);
```

Comment:
If the classifier consists of more than one word, the array will contain more than two strings. For example: 
given
```
<subfield code="a"> 9781410352224 (some qualifier)</subfield>
```
result
```
getProductId - 9781410352224 
qualifier - (some
```
expected result
```
getProductId - 9781410352224 
qualifier - (some qualifier)
```

Reply:
Done, added length to split function